### ISSUE TITLE
Calling Interface functions in main thread from non-blocking Server

### ISSUE THREAD

**Issue by munkybutt** [wgnet/clacks#1]
Hey - I am attempting to use clacks to communicate between two instances of 3DsMax.

If I start the server in blocking mode, the 3DsMax instance hosting the server locks.
```python
# Max locks:
server.start(blocking=True)
```
If I start it in non-blocking, then any max commands I call in my interface error out as you can only run them in the main thread.
```python
# Max functions error:
server.start(blocking=False)
```

Is there a way to have my interface functions be called in the main thread from a non-blocking server?

**Comment by MaVCArt** [comment]
Yes - you will need to use clacks_qt for this :) it was specifically designed for this purpose

**Comment by MaVCArt** [comment]
Worth noting that in its vanilla state, clacks will do very little for you - the real power lies in the extension libraries

**Comment by MaVCArt** [comment]
https://github.com/MaVCArt/clacks_qt
